report/test/controllers/ReportIntegrationSpec.scala

```diff
@@ -0,0 +1,99 @@
+package controllers
+
+import models._
+import org.joda.time.{DateTimeZone, DateTime}
+import org.specs2.mock.Mockito
+import org.specs2.specification.Scope
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.inject.bind
+import play.api.test._
+import services.{NotificationReportRepositorySupport, Configuration}
+import tracking.InMemoryNotificationReportRepository
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ReportIntegrationSpec extends PlaySpecification with Mockito {
+
+  trait ReportTestScope extends Scope {
```

Comment:
I'd move test scope to the bottom, test definitions are more important than setup